doc/intro.rst

```diff
@@ -293,6 +293,37 @@ To run a L2L simulation, copy the file :file:`bin/l2l-template.py` (see :doc:`l2
 initialization of the appropriate `Optimizers` and `Optimizees`. The rest of the code should be left in place for
 logging and recording. See the source of :file:`bin/l2l-template.py` for more details.
 
+Execution setup
+~~~~~~~~~~~~~~~
+The L2L framework works with JUBE in order to deploy the execution of the different instances of the optimizee on
+the available computational resources. This requires that the trajectory contains a parameter group called JUBE_params
+which contains details for the right execution of the program.
+
+**Mandatory** steps to define the execution of the optimizees:
+1. Add a parameter group to the :obj: traj called JUBE_params using its :meth: f_add_parameter_group.
```

Comment:
All these three steps probably needs more details that "using f_add_parameter_group" since `f_add_parameter_group` is a very generic function -- specifically in step 2. and 3., what parameters need to be added?